### ISSUE TITLE
Add sample outputs for each query.

### ISSUE THREAD

**Issue by ssrihari** [nilenso/postgresql-monitoring#1]

**Comment by shrayasr** [comment]
Sample outputs can clutter the README even more (#2). Probably a better thing to do is to describe the columns in the output?

**Comment by ssrihari** [comment]
@shrayasr True. Will try to find a representation that doesn't compromise readability while presenting information that's necessary. A PR would be more than welcome though :)

**Comment by shrayasr** [comment]
> Will try to find a representation that doesn't compromise readability while presenting information that's necessary. 

Meaning we'll keep the output in a non cluttery format somehow?

> A PR would be more than welcome though :)

Would be glad to once we finalize what it is we're going to put in :blush:

**Comment by shrayasr** [comment]
Here's a thought: Can't we keep the details on the wiki and keep a short and sweet version on the README?
